src/elements/content-uploader/UploadState.scss

```diff
@@ -56,14 +56,6 @@
     }
 }
 
-.bcu-upload-state-message + .be-input-btn {
-    margin-top: 10px;
-}
-
-.bcu-upload-state-message + .be-input-link {
-    margin-top: 5px;
-}
```

Comment:
unused styles from a previous design iteration. these styles are no longer applied since `bcu-upload-input-container` is now the sibling of `bcu-upload-state-message` instead of `be-input-btn` or `be-input-link`